packages/menu/src/Menu.tsx

```diff
@@ -98,7 +112,7 @@ function Menu({
 
     function updateChildren(children: React.ReactNode): React.ReactNode {
       return React.Children.map(children, child => {
-        if (!React.isValidElement(child) || child.props?.disabled) {
+        if (!React.isValidElement(child)) {
```

Comment:
Should we highlight this change as well?

Reply:
i'll add it

Reply:
Actually i'm going to add this back. If it's disabled it doesn't need to be cloned or have any additional props added to it.